### DISCUSSION TITLE
redirect 1 domain and subdomains to other NPM

### DISCUSSION THREAD

**Discussion by GerritKuilder** [NginxProxyManager/nginx-proxy-manager#3064] [Q&A]
I have a few domains and would like to handle all the redirects for 1 domain (including all the subdomains) by another NPM than my main one).
Internet -> home router -> NPM1
On the external DNS server I have a wildcard setup (*.X.com) pointing to my external IP., all working as expected.
I redirect all requests for port 443 and 80 to my NPM1 running on my main server, but would like to redirect all requests for domain X.com and it's subdomains to  NPM2 running on another server.

Is this possible or do I have to do some DNS stuf on my router or the server where NPM1 is running on.

Any tips appreciated.

Kind Regards,

Gerrit Kuilder

**Comment by samypan** [comment]
Did you end up finding the solution? Currently on the same boat.
